snippets/open-yield-block-({).sublime-snippet

```diff
@@ -1,6 +0,0 @@
-<snippet>
-    <content><![CDATA[{ ${1/(^(?<var>\s*[a-z_][a-zA-Z0-9_]*\s*)(,\g<var>)*,?\s*$)|.*/(?1:|)/}${1:variable}${1/(^(?<var>\s*[a-z_][a-zA-Z0-9_]*\s*)(,\g<var>)*,?\s*$)|.*/(?1:| )/}${2:$TM_SELECTED_TEXT} ]]></content>
-    <tabTrigger>{</tabTrigger>
-    <scope>source.crystal - string - comment</scope>
-    <description>Insert { |variable| … }</description>
-</snippet>
```

Comment:
Could you please elaborate on an example of what was this snippet doing? I cannot get it from the code...